pegasus_wrapper/workflow.py

```diff
@@ -67,6 +71,21 @@ class WorkflowBuilder:
     in the workflow graph. We compute this based on a job signature
     and only actually schedule the job once.
     """
+    _added_files: Set[File] = attrib(init=False, factory=set)
```

Comment:
Please add comments to this field and the following one explaining what they are for.